examples/basics/export_data.ipynb

```diff
@@ -31,7 +31,7 @@
       "metadata": {},
```

Comment:
<p>This is giving me </p><p>Have you confirm this works as expected ? </p><pre class="ql-syntax" spellcheck="false">file size:  None
line count:  None
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>